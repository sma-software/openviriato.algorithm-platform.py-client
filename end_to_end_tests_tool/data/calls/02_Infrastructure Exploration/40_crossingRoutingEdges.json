{
    "restResource": "/nodes/{id}/crossing-routing-edges",
    "httpMethod": "GET",
    "urlParameters": [ "281" ],
    "requestBody": null,
    "CSharpMethodName": "GetCrossingRoutingEdges(routingPoint)",
    "CSharpMethodParameterTypes": [ "SMA.AlgorithmPlatform.AlgorithmInterface.AIDM.RoutingPoint" ],
    "subcategory": "Get Crossing Routing Edges"
}
