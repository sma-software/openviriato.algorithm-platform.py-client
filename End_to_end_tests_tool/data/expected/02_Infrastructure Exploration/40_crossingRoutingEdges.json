{
  "RoutingEdges":
  [
    {
      "StartSectionTrackID":887,
      "EndSectionTrackID":888,
      "NodeID":281
    },
    {
      "StartSectionTrackID":888,
      "EndSectionTrackID":887,
      "NodeID":281
    }
  ]
}